src/measure_theory/function/lp_space.lean

```diff
@@ -2350,8 +2455,8 @@ lemma smul_comp_LpL [fact (1 ≤ p)] (c : 𝕜) (L : E →L[𝕜] F) :
 by { ext1 f, exact smul_comp_Lp c L f }
 
 /-- TODO: written in an "apply" way because of a missing `has_smul` instance. -/
```

Comment:
is the TODO still relevant?

Reply:
Sadly, yes. I'll have a look again later today and update the comment with more detail

Reply:
I take it back; I was able to remove it after all by adding the missing `smul_comm_class` instance!